Jellyfin.Server/Migrations/Routines/MigrateActivityLogDb.cs

```diff
@@ -56,9 +80,11 @@ public void Perform()
                     var newEntry = new ActivityLog(
                         entry[1].ToString(),
                         entry[4].ToString(),
-                        entry[6].SQLiteType == SQLiteType.Null ? Guid.Empty : Guid.Parse(entry[6].ToString()),
-                        entry[7].ReadDateTime(),
-                        ParseLogLevel(entry[8].ToString()));
+                        entry[6].SQLiteType == SQLiteType.Null ? Guid.Empty : Guid.Parse(entry[6].ToString()))
+                    {
+                        DateCreated = entry[7].ReadDateTime(),
+                        LogSeverity = logLevelDictionary[entry[8].ToString()]
```

Comment:
we should handle a case when `entry[8].ToString()` is not in a dictionary, i.e. make it default to `Trace` if nothing fits.